src/App/routes/Connections/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { ibc } from "@cosmjs/stargate/types/codec";
+import React, { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+
+import { useClient } from "../../../contexts/ClientContext";
+import { IbcConnectionsResponse } from "../../../types/ibc";
+import { ellideMiddle } from "../../../utils/strings";
+import { HeightData } from "../../components/HeightData";
+import { Navigation } from "../../components/Navigation";
+import { pathConnection } from "../../paths";
+import { style } from "../../style";
+import { SelectClientId } from "./SelectClientId";
+
+interface ConnectionsParams {
+  readonly clientId: string;
+}
+
+export function Connections(): JSX.Element {
+  const { clientId } = useParams<ConnectionsParams>();
+  const { getClient } = useClient();
+
+  const [connectionsResponse, setConnectionsResponse] = useState(new IbcConnectionsResponse());
+
+  useEffect(() => {
+    (async function updateConnectionsResponse() {
+      const connectionsResponse = new IbcConnectionsResponse(await getClient().ibc.unverified.connections());
+
+      if (clientId) {
+        try {
+          const { connectionPaths } = await getClient().ibc.unverified.clientConnections(clientId);
+          const connections = connectionPaths
+            ? await Promise.all(
+                connectionPaths.map(async (connectionId) => {
+                  const responsePromise = await getClient().ibc.unverified.connection(connectionId);
```

Comment:
This is a slightly misleading name since there's an `await` here. `connectionResponse`?